app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,7 +25,7 @@ export default function NoteTable({
 
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
```

Comment:
Please add `nui-table--hover` class too. 
It adds the hover effects for the table rows.

Reply:
I have added the nui table hover class.